queries/shorts.sql

```diff
@@ -12,6 +12,25 @@ WHERE s.status = 'published'
 GROUP BY week
 ORDER BY week DESC;
 
+-- name: ListSegmentedShortIDsForRolesWithScores :many
+SELECT s.id,
+       -- We need a date and if we do not have a published_at date, we need to assume that the created date is when it was published
+       EXTRACT(DAY FROM current_date - COALESCE(mi.published_at, mi.date_created)) age,
+
+       -- For 10 days the shorts is boosted. It starts with a 5 points boost, and the boost "degrades" by 0.5
```

Comment:
boosting it by 5 points, means that the first 2 days, most shorts shown will be the new ones. That has the nice side-effect that we get feedback about the short a lot quicker than before because ~150 people will have seen the short after 1 day.